examples/vue-solana/src/App.vue

```diff
@@ -1,7 +1,7 @@
 <script lang="ts" setup>
 import {
   createWeb3Modal,
-  defaultConfig,
+  defaultSolanaConfig,
```

Comment:
There was no export of `defaultConfig` from the solane/vue package. The correct import is `defaultSolanaConfig`

Mostly mirrored the set up from `examples/react-solana`